modules/beta-private-cluster/variables.tf

```diff
@@ -393,6 +393,14 @@ variable "master_ipv4_cidr_block" {
   default     = "10.0.0.0/28"
 }
 
+variable "master_global_access_enabled" {
```

Comment:
Can we expose this via the [safer-cluster](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/autogen/safer-cluster) too?

Reply:
Actually let's not over-parameterize. Let's just set it to true on the safer cluster.

Reply:
ok sounds good